angular/src/app/footer/footer.component.html

```diff
@@ -0,0 +1,30 @@
+<div id="footer" class="container valign-wrapper">
+  <div class="valign-wrapper">
+    <a href="https://www.twitch.tv/beginnerstaikotournament" target="_blank"><img src="./assets/images/twitchlogo.png"
+        alt="twitter"></a>
+    <a href="https://www.twitch.tv/beginnerstaikotournament" id="twitch" target="_blank">Twitch</a>
+  </div>
+
+  <div class="valign-wrapper">
+    <a href="https://challonge.com/BTTS5" target="_blank"><img src="./assets/images/challongelogo.png"
+        alt="challonge"></a>
+    <a href="https://challonge.com/BTTS5" id="challonge" target="_blank">Challonge</a>
+  </div>
+
+  <div class="valign-wrapper">
+    <a href="#" id="legal" target="_blank">Legal Notice</a>
+  </div>
+
+  <div class="valign-wrapper">
+    <a href="https://discord.gg/EYAqSWH" target="_blank"><img src="./assets/images/discordlogo.png" alt="discord"></a>
+    <a href="https://discord.gg/EYAqSWH" id="discord" target="_blank">Discord</a>
+  </div>
+
+  <div class="valign-wrapper">
+    <a href="https://osu.ppy.sh/community/forums/topics/994702" target="_blank"><img src="./assets/images/osu.png" alt="forum post"></a>
```

Comment:
currently, these icons aren't aligned correctly; they're slightly off-center
consider applying `vertical-align: middle` to these img